langchain/src/chains/api/prompts.ts

```diff
@@ -3,11 +3,23 @@ import { PromptTemplate } from "../../prompts/prompt.js";
 
 export const API_URL_RAW_PROMPT_TEMPLATE = `You are given the below API Documentation:
 {api_docs}
-Using this documentation, generate the full API url to call for answering the user question.
-You should build the API url in order to get a response that is as short as possible, while still getting the necessary information to answer the question. Pay attention to deliberately exclude any unnecessary pieces of data in the API call.
 
-Question:{question}
-API url:`;
+<< FORMATTING >>
+Return a JSON string with a JSON object formatted to look like:
```

Comment:
Can we use an output fixing parser here?

https://js.langchain.com/docs/modules/prompts/output_parsers/#output-fixing-parser

Reply:
Yes will make use of it @jacoblee93

Reply:
Can you try replacing this formatting instructions with the built-in ones for your output parser?

https://js.langchain.com/docs/modules/prompts/output_parsers/#structured-output-parser-with-zod-schema

```
const parser = StructuredOutputParser.fromZodSchema(
  z.object({
    answer: z.string().describe("answer to the user's question"),
    sources: z
      .array(z.string())
      .describe("sources used to answer the question, should be websites."),
  })
);

const formatInstructions = parser.getFormatInstructions();

const prompt = new PromptTemplate({
  template:
    "Answer the users question as best as possible.\n{format_instructions}\n{question}",
  inputVariables: ["question"],
  partialVariables: { format_instructions: formatInstructions },
});
```